schema_utils.py

```diff
@@ -447,3 +448,15 @@ def is_valid_math_equation(obj):
         if lhs_is_numerically_valid and rhs_is_algebraically_valid:
             return True
         return False
+
+    @staticmethod
+    def is_supported_audio_language_code(obj):
```

Comment:
change obj to languageCode

Reply:
Done

Reply:
@DubeySandeep this change was reverted back to `obj`. `language_code` was causing a validation test to fail (```ConfigPropertyRegistryTests.test_config_property_schemas_are_valid```)

Reply:
I see, thanks for the info!